src/payment/OrderSummary.jsx

```diff
@@ -0,0 +1,242 @@
+/* eslint-disable react/style-prop-object */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { FormattedNumber, FormattedMessage } from '@edx/frontend-i18n';
+
+import { basketSelector } from './data/selectors';
+import { CouponForm } from './coupon';
+
+import { ORDER_TYPES } from './data/constants';
+
+function SummaryTable({ price }) {
+  if (price === undefined) return null;
+
+  return (
+    <div className="summary-row d-flex">
+      <span className="flex-grow-1">
+        <FormattedMessage
+          id="payment.summary.table.label.price"
+          defaultMessage="Price"
+          description="Label for price excluding discount line on order summary table"
```

Comment:
I don't think we need to specify "excluding discount line"-- I think that makes it hard for a non-native English speaker  to understand, and probably all they need to know is that it's the price.